specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -48,6 +48,12 @@ One example is features that were cut during scoping.
 
 ## High-level design
 
+#### Folder Structure
+
+* [frontend-ui](../../projects/frontend-ui) The root folder for all frontend code.
+  * [shared-components](../../projects/frontend-ui/shared-components/README.md): the root folder for the shared components project, for more details please see the readme.
```

Comment:
suggestion: drop "-components" suffix, since a loot of directories are components so -components is actually redundant.

Reply:
or maybe shared-components is good; it was like that before, right?

Reply:
yeah it was like that before. 
I will revert.